aviary/examples/external_subsystems/engine_NPSS/NPSS_Model/Output/turbojet.viewOut

```diff
@@ -0,0 +1,41 @@
+************************************************************************************************************************************
+Date:  ����1V    Time:  ����1V    Model:                                                                    converge = 1   CASE:   0 
```

Comment:
more misinterpreted characters?

Reply:
These commands don't seem to work in my NPSS linux build.  I removed the offending items from the file and the file that creates this file.